client/app/lib/components/layouts/Subsection.tsx

```diff
@@ -0,0 +1,35 @@
+import { Typography } from '@mui/material';
+import { ReactNode } from 'react';
+
+interface SubsectionProps {
+  title?: string;
+  subtitle?: string;
+  children?: ReactNode;
```

Comment:
I originally left `title` and `subtitle` optional because one may only want to add `subtitle` without `title`, and vice versa.

To align with `Section`, I will make `title` required.